docs/config/environment.rst

```diff
@@ -110,7 +110,7 @@ General
 
   - For `basic installations`_, the process working directory
 
-  - If you have installed `a CrateDB Linux package`_, ``/var/lib/crate``
+  - If you have installed a CrateDB Linux package, ``/var/lib/crate``
```

Comment:
The new installation tutorial has two sections for CrateDB Linux package installation:
* https://crate.io/docs/crate/tutorials/en/latest/install.html#debian-and-ubuntu
* https://crate.io/docs/crate/tutorials/en/latest/install.html#red-hat-linux-and-centos

This makes it a bit complicated to link to general Linux package. I removed the link for now as I wasn't sure what would be best here.

Any suggestion where to link to?

Reply:
Yeah. While we now have everything on a single page, canonically linking to "Linux" is currently not possible. It's a balancing act.

For me, it is either fine to keep it like it is now or otherwise improve this detail again. Maybe @proddata and @mfussenegger want to add some thoughts here.

Reply:
We could create a heading hierarchy. 

```
# Install via Linux package

## Debian or Ubuntu

## Redhat or Centos
```

Reply:
Sure, I can do that if linking there would be more convenient.

Reply:
Implemented by https://github.com/crate/crate-tutorials/pull/53.

Reply:
You can now link to the Linux section by using https://crate.io/docs/crate/tutorials/en/latest/install.html#linux.